### PR TITLE
Make it compatible with @ember/test-helpers and ember-qunit v5

### LINKED ISSUE #524: Errors running tests since upgrading to v5 of `ember-qunit`
Migrating to v5 of ember-qunit causes issues with ember-cli-page-object. I suspect it is to do with the following [breaking change](https://github.com/emberjs/ember-qunit/blob/master/docs/migration.md#remove-ember-test-helpers-modules)

Actual error: 
```
  Error: Could not find module `ember-test-helpers/wait` imported from `ember-cli-page-object/test-support/-private/execution_context/integration`
```

### PR DESCRIPTION
**Pull request by SergeAstapov**
Fixes #524

### CODE REVIEW

**Comment by ro0gr** [comment]
Awesome 👏 Can we also add an ember-try scenario, to make sure it works with the ember-qunit@5?

**Comment by SergeAstapov** [comment]
@ro0gr sure! That makes sense. I'll update the PR

**Comment by SergeAstapov** [comment]
also I plan to create similar PR in ember-native-dom-helpers

**Comment by SergeAstapov** [comment]
@ro0gr I didn't succeed with ember-try scenario, could not get tests running with ember-qunit v5. Do you mind to merge it "as is"?

**Comment by ro0gr** [comment]
@SergeAstapov alright, I'd like to take a look on ember-try scenario, or at least test it manually, before merging. Going to give it a try this weekend.

**Comment by ro0gr** [comment]
@SergeAstapov I've just gave it a try, and seems fixing it in scope of "ec-page-object" isn't sufficient, unfortunately.

"ember-native-dom-helpers" imports `wait` helper from the "ember-test-helpers", which is not a thing in the "@ember/test-helpers" anymore, so the test suite here fails immediately.

I hope your PR(cibernox/ember-native-dom-helpers#168) to the native-dom-helpers would be solved soon, so v1 can start supporting ember-qunit@5.

**Comment by ro0gr** [comment]
This continued and finished in https://github.com/san650/ember-cli-page-object/pull/530. Thanks!

File: addon-test-support/-private/compatibility.js [hunk 1]
```diff
@@ -57,6 +57,8 @@ if (require.has('ember-test-helpers/wait')) {
   // conditions `ember-test-helpers/wait` can still be in the pending state
   // at this point, so its exports are still undefined.
   waitFn = (...args) => require('ember-test-helpers/wait').default(...args);
+} else if (require.has('@ember-test/helpers')) {
+  waitFn = (...args) => require('@ember-test/helpers').wait(...args);
```
Comment (NullVoxPopuli) [inline addon-test-support/-private/compatibility.js:61]: shouldn't this be `@ember/test-helpers` not `@ember-test/helpers`?

https://github.com/emberjs/ember-test-helpers/blob/master/API.md#waitfor

`@ember-test/helpers` doesn't exist:
image

Comment (ro0gr) [reply addon-test-support/-private/compatibility.js:61]: It was fixed in a follow up https://github.com/san650/ember-cli-page-object/commit/75a9e8b2dff49b4c513e61422215c5ca168b33d5#diff-d52aad060d2deba9d04ba86013c19734ea5e79e60b7539d2bd563b561288e7b3

**Pull request closed without merging**